### DISCUSSION TITLE
DigitalCore IRC network error

### DISCUSSION THREAD

**Discussion by speedster-pta** [autobrr/autobrr#1044] [Q&A] [answered]
I am not able to get Autobrr to connect with Digitalcore's IRC network. I have 5 other indexers all working, but can't the digitalcore settings to work. My settings are as follows:

Name: Digitalcore
Enabled: yes
Server: irc.digitalcore.club
Port: 7000
TLS: yes
Password: blank
Nick: user_autodl
Mechanism: none
Invite command: /msg ENDOR !invite user_autodl irc_key
Channels: #announce

Is anyone perhaps able to see my error?

**Comment by s0up4200** [comment, marked as answer]
![image](https://github.com/autobrr/autobrr/assets/18177310/e37fcff4-7efc-41e1-8764-fdbfcd1ff7cc)
You should probably put your Digitalcore username here, not your irc nick. And seems you maybe forgot your irc key?
Also, remove `/msg` from your invite command.

**Reply by speedster-pta** [reply]
Awesome. Thank! It was the username.
